src/app/logic/Players/ComputerPlayer/ComputerPlayer.js

```diff
@@ -0,0 +1,26 @@
+function createPlayer() {
+    const player = {
+        giveAnswerCallbacks: [],
+
+        onGiveAnswerDo(callback) {
+            this.giveAnswerCallbacks.push(callback);
+        },
```

Comment:
I'm not sure if this is the best approach. If we would have like 5 callbacks here, we would have to call this function 5 times. Why not make it something like this:

> onGiveAnswerDo(callbacks) {
    this.giveAnswerCallbacks = callbacks;
}

It would accept an array of callbacks instead of just one callback

Reply:
Good idea, I modified the code so that it works with your suggestion.